### ISSUE TITLE
Gamemode 3

### ISSUE THREAD

**Issue by renzotom** [EssentialsX/Essentials#689]
When is player in Gamemode 3 and gold Shift (crouch), he start fall down into void hole.

**Comment by SupaHam** [comment]
What version of Spigot and EssentialsX are you using?

**Comment by renzotom** [comment]
I use newest version of spigot 1.10.2 and newest version of EssentialsX, but when I had 1.10, it was also buggy :/ .

**Comment by Kakifrucht** [comment]
Why would that be weird? Shifting while in gamemode 3 makes you fly down.

**Comment by SupaHam** [comment]
Please don't say latest version, I am asking for the exact versions. You can type `/ver` to get the spigot version and `/ess reload` to get the essentials version.

**Comment by drtshock** [comment]
This is a minecraft _feature_

**Comment by renzotom** [comment]
But, I fall down into void and when I stop hold shift, I still fall down. This is 100% bug, it wasn´t in 1.9 version, maybe it is bug in essentialsX. I can record video for you, if you want.  My version of spigot is:
gif-Spigot-897a019-5047354 (MC: 1.10.2) (Implementing API version 1.10.2-RO.1-SNAPSHOT).
You are 9 version(s) behind (PS: I go update :D)
My version of EssentialsX is 2.0.1-b366

**Comment by SupaHam** [comment]
I would ask that you get back to us after you have updated spigot and confirmed that it is still an issue.

**Comment by renzotom** [comment]
Now, I running on latest version (this is in /ver) and when I go under ground, I start fall down to void... I think, this is bug in spigot or here in essentials. But, I don´t fall, when I am in air or above ground, but when I am under ground I fall down to big whole void and dead :( .

**Comment by drtshock** [comment]
Can you reproduce this without essentials installed?

**Comment by renzotom** [comment]
Oh really, problem with gamemode is mistake in Minecraft or bukkit. But what problem with fly? And sorry for this.

**Comment by SupaHam** [comment]
I honestly don't understand anymore. You're saying this issue is in Minecraft but then you ask what the problem is? Can you please say as much as you can about your issue, I currently find it hard to follow.

**Comment by renzotom** [comment]
I think, what is problem in Fly, I mean here https://github.com/drtshock/Essentials/issues/691 (oh, I forgot sent log, wait please). I know, here is problem in Minecraft. You can close this issue.
